### PR TITLE
Add line to enable PostgreSQL 12 module on EL8

### PR DESCRIPTION
**Pull request by ehelms**
Cherry-pick into:

* [x] Foreman 2.4 / Katello 4.0

### CODE REVIEW

**Comment by ehelms** [comment]
CC @ekohl

**Comment by ehelms** [comment]
CC @ekohl

File: guides/common/modules/proc_configuring-repositories.adoc [hunk 1]
```diff
@@ -58,6 +58,15 @@ ifdef::katello[]
 # {package-manager} config-manager --set-enabled powertools
 ----
 endif::[]
+ifdef::foreman-el,katello[]
```
Comment (ekohl) [inline guides/common/modules/proc_configuring-repositories.adoc:61]: I'm not sure if this needs a guard on katello. IMHO this is valid for vanilla Foreman on EL8 as well.

File: guides/common/modules/proc_configuring-repositories.adoc [hunk 2]
```diff
@@ -58,6 +58,15 @@ ifdef::katello[]
...
++
+. Enable the PostgreSQL 12 module:
```
Comment (ekohl) [inline guides/common/modules/proc_configuring-repositories.adoc:63]: I wonder if we need some reference documentation on what to do if a different module is already enabled etc.

File: guides/common/modules/proc_configuring-repositories.adoc (hunk 1 above)
Comment (ehelms) [reply guides/common/modules/proc_configuring-repositories.adoc:61]: That's not a guard, but rather a "which guides to include this in" line. So in this case, include it in the Foreman Enterprise Linux guide and the Katello guide.

File: guides/common/modules/proc_configuring-repositories.adoc (hunk 2 above)
Comment (ehelms) [reply guides/common/modules/proc_configuring-repositories.adoc:63]: Added a note

**Comment by lzap** [comment]
I am just about to branch off 2.4 series, this is something that should go into 2.4/4.0. Can we merge now?

**Comment by ehelms** [comment]
> I am just about to branch off 2.4 series, this is something that should go into 2.4/4.0. Can we merge now?

This applies only to nightly and Foreman 2.5+.

**Comment by lzap** [comment]
Thank you!

File: guides/common/modules/proc_configuring-repositories.adoc (hunk 1 above)
Comment (ekohl) [reply guides/common/modules/proc_configuring-repositories.adoc:61]: So I read `ifdef` as an AND, not an OR. IMHO this should be an OR. If it is, then consider this addressed.

**Review by ekohl — APPROVED** [review]
If `ifdef` works as an OR then :+1: on this.

Since this is the setup guide, I think it wouldn't be bad to do this even on a released version (2.4). Upgrading users shouldn't read it. Plus, the whole docs.theforeman.org is still considered in development. Regular Foreman users are probably not reading it as a primary source yet.

**Comment by lzap** [comment]
> If `ifdef` works as an OR then 👍 on this.

That is my understanding as well but frankly I haven't tried to verify this myself.

**Comment by ehelms** [comment]
> If `ifdef` works as an OR then  on this.

`ifdef` can work as AND or OR:

OR:
```
ifdef::foreman-el,katello[]
```

AND:
```
ifdef::foreman-el+katello[]
```

Ref: https://docs.asciidoctor.org/asciidoc/latest/directives/ifdef-ifndef/#checking-multiple-attributes

**Comment by melcorr** [comment]
Can I merge this?

**Comment by ekohl** [comment]
I think it can also be cherry picked into the 2.4 branch. @ehelms?

**Comment by ehelms** [comment]
> I think it can also be cherry picked into the 2.4 branch. @ehelms?

Yes, it can apply there as well

**Comment by ekohl** [comment]
How do we deal with cherry picks in this repo? Is that a separate PR? If so, who's supposed to create it?

**Comment by melcorr** [comment]
Me - I'll do it !

**Pull request merged**